docs/source/user guide/controller configuration.rst

```diff
@@ -28,22 +28,22 @@ whether or not your relay requires the pin to be inverted (for most mechanical r
     you have an actual door sensor connected to your controller. If you set this up incorrectly your controller may think
     the door to your fermentation chamber is always open, and fail to trigger heating/cooling as expected.
 
-
 Temperature Sensors
 ---------------------------
 
-OneWire temperature sensors are configured by device address rather than the pin they are connected to. Connect your
-OneWire sensors to your BrewPi controller, wait a few seconds, and refresh the page. You should see all connected
-sensors listed under either Available Devices or Installed Devices.
+Fermentrack supports the DS18B20 temperature sensors, which use the OneWire protocol. This allows multiple sensors to be connected via a single pin on the controller board. For the ESP8622 board, this is the D6 pin. To ensure stable data transfers, you will also want to include a 4.7K resistor between the voltage source and the data pin. An example of how to gang multiple temperature sensors together, with resistor, looks like this:
 
-To install an available temperature sensor, simply choose the device function (Chamber Temp, Room Temp, Beer Temp,
-etc.) and click "Assign". Your BrewPi controller will configure the device appropriately.
+.. image:: https://www.microcontroller-project.com/uploads/2/2/1/5/22159166/nodemcu-esp8266-with-multiple-ds18b20-sensors_orig.png
```

Comment:
A couple of notes about adding this in -

I like the idea of adding pictures to the documentation wherever possible, but if it can be done I’d prefer to self-host rather than to incorporate external links. For this specific diagram, I like how it shows the wiring for chained DS19B20 sensors, but I think that it could be confusing considering we don’t know the device/pin the user is using. I would edit the image to just show the generic pins that are used for every build (ground, VCC, data) rather than the device those wires connect to.

Reply:
Easy enough to add the pic to the repo, I wasn't sure your thought on that... it was just something I pulled off Google, it may be better to make one from scratch anyway. 

The question I have, and am unable to find a documented answer for- is there a dedicated pin for OneWire on the ESP? Back on HBT a few years back you posted a pinout indicating pin 6 was used for OneWire- is that hard-coded somewhere, or is any pin able to be used? I would love to be able to answer that question definitively here.

Reply:
Yep, D6. The pin out here is accurate: https://github.com/thorrak/brewpi-esp8266#expected-pinout-note---changed-in-v03

That said, I would again reference my comment below - I would leave out any build-specific wiring references as the pins are different for each controller (ESP8266, Arduino, ESP32, Fuscus, etc.)

Reply:
Done